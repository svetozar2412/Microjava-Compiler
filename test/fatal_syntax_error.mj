program FatalSyntaxError
	{
		void main()
		{
			print(++++);
		}
	}