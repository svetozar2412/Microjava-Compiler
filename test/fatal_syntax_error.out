INFO  11:01:25,031 - Kompajliranje src fajla: C:\Users\Svetozar Divljakovic\eclipse-workspace\MJCompiler\src\..\test\fatal_syntax_error.mj
INFO  11:01:25,068 - #21 program
INFO  11:01:25,068 - #46 FatalSyntaxError
INFO  11:01:25,068 - #22 {
INFO  11:01:25,069 - #41 void
INFO  11:01:25,069 - #46 main
INFO  11:01:25,070 - #25 (
INFO  11:01:25,070 - #26 )
INFO  11:01:25,071 - #22 {
INFO  11:01:25,071 - #40 print
INFO  11:01:25,071 - #25 (
INFO  11:01:25,071 - #36 ++
ERROR 11:01:25,071 - 
Sintaksna greska na liniji 5
INFO  11:01:25,072 - #36 ++
INFO  11:01:25,072 - #26 )
INFO  11:01:25,082 - #24 ;
INFO  11:01:25,082 - #23 }
INFO  11:01:25,083 - #23 }
ERROR 11:01:25,083 - Fatalna greska, parsiranje se ne moze nastaviti na liniji 7
ERROR 11:01:25,083 - Parsiranje NIJE izvrseno uspesno!
